views/home.hbs

```diff
@@ -0,0 +1,23 @@
+<div id="addon">
+    <img class="logo" src="{{manifest.logo}}">
+    <h3 class="version"><i>{{manifest.version}}</i></h3>
+    <h3 class="description">{{manifest.description}}</h3>
+
+    <h2 class="gives">This add-on has:</h2>
+    <ul>
+        <li>
+            More
+            {{#each manifest.types}}
+                {{stylize this}}
+            {{/each}}
+            in Discover
+        </li>
+        <li>Watch for FREE</li>
+    </ul>
+
+    <a class="activate-link" href="stremio://{{manifest.id}}/stremioget/stremio/v1"><button name="Install">Install Add-on</button></a>
+    <h3 class="contact">
+        To contact add-on creator:
+        <a href="mailto:">{{manifest.contactEmail}}</a>
```

Comment:
I don't think this is correct, it will show a link, the link text will be the email, but the link itself will still be `mailto:` which leads nowhere

we should also think of cases in which the `contactEmail` is not set at all (as it's optional in the docs), and this link shouldn't appear in such cases